backbone/src/main/java/org/researchstack/backbone/ui/step/layout/ConsentVisualStepLayout.java

```diff
@@ -91,6 +91,11 @@ public void isEditView(boolean isEditView) {
         // no-op: Only needed when the user is on edit mode inside regular steps
     }
 
+    @Override
+    public StepResult getStepResult() {
+        return null;
```

Comment:
In this (and all here and in Axon) that return null, do you mind leaving a comment briefly explaining why? (see slack for more info). Thanks! :)